src/stories/GifPicker.stories.tsx

```diff
@@ -82,3 +106,11 @@ export const Trending = {
 		await userEvent.click(canvas.getAllByTestId('gpr-category')[0]);
 	}
 };
+
+export const InitialSearchTerm = {
+	...Home,
+	args: {
+		...Home.args,
+		initialSearchTerm: 'love'
+	}
+};
```

Comment:
We already have `Search` story and that's just a duplicate. Please change `Search` story to use `initialSearchTerm` instead of user event rather than create new one